[2024-11-17T23:15:11.011+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-17T23:15:11.045+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: capstone.get_request_api manual__2024-11-17T23:15:06.122665+00:00 [queued]>
[2024-11-17T23:15:11.062+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: capstone.get_request_api manual__2024-11-17T23:15:06.122665+00:00 [queued]>
[2024-11-17T23:15:11.064+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2024-11-17T23:15:11.092+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_request_api> on 2024-11-17 23:15:06.122665+00:00
[2024-11-17T23:15:11.106+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1418) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-17T23:15:11.114+0000] {standard_task_runner.py:72} INFO - Started process 1420 to run task
[2024-11-17T23:15:11.115+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'capstone', 'get_request_api', 'manual__2024-11-17T23:15:06.122665+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/capstone_***/country.py', '--cfg-path', '/tmp/tmp_tad8srf']
[2024-11-17T23:15:11.119+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask get_request_api
[2024-11-17T23:15:11.206+0000] {task_command.py:467} INFO - Running <TaskInstance: capstone.get_request_api manual__2024-11-17T23:15:06.122665+00:00 [running]> on host ea8ac57aed30
[2024-11-17T23:15:11.427+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='adewunmi' AIRFLOW_CTX_DAG_ID='capstone' AIRFLOW_CTX_TASK_ID='get_request_api' AIRFLOW_CTX_EXECUTION_DATE='2024-11-17T23:15:06.122665+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-17T23:15:06.122665+00:00'
[2024-11-17T23:15:11.433+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-17T23:15:11.487+0000] {cap.py:24} INFO - GET request initiating
[2024-11-17T23:15:13.304+0000] {cap.py:29} INFO - API request connection is okay
[2024-11-17T23:15:13.437+0000] {cap.py:35} INFO - Response data received successfully
[2024-11-17T23:15:13.440+0000] {python.py:240} INFO - Done. Returned value was:        tld cca2  ... currencies.NIO.name currencies.NIO.symbol
0    [.gs]   GS  ...                 NaN                   NaN
1    [.gd]   GD  ...                 NaN                   NaN
2    [.ch]   CH  ...                 NaN                   NaN
3    [.sl]   SL  ...                 NaN                   NaN
4    [.hu]   HU  ...                 NaN                   NaN
..     ...  ...  ...                 ...                   ...
245  [.be]   BE  ...                 NaN                   NaN
246  [.il]   IL  ...                 NaN                   NaN
247  [.nz]   NZ  ...                 NaN                   NaN
248  [.ni]   NI  ...  Nicaraguan c√≥rdoba                    C$
249  [.ai]   AI  ...                 NaN                   NaN

[250 rows x 896 columns]
[2024-11-17T23:15:14.068+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-17T23:15:14.069+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=capstone, task_id=get_request_api, run_id=manual__2024-11-17T23:15:06.122665+00:00, execution_date=20241117T231506, start_date=20241117T231511, end_date=20241117T231514
[2024-11-17T23:15:14.112+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-17T23:15:14.158+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-17T23:15:14.169+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
